### PR TITLE
remove too old note from "Garbage Collection"

### PR DESCRIPTION
**Pull request by makocchi-git**

### CODE REVIEW

**Comment by makocchi-git** [comment]
[check url is here](https://deploy-preview-9444--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/controllers/garbage-collection/)
[original](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/)

File: content/en/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ weight: 60
 The role of the Kubernetes garbage collector is to delete certain objects
 that once had an owner, but no longer have an owner.
 
+{{< note >}}
 **Note**: Garbage collection is a beta feature and is enabled by default in
 Kubernetes version 1.4 and later.
+{{< /note >}}
```
[the remark this answers is not in the document]
Comment (makocchi-git) [reply content/en/docs/concepts/workloads/controllers/garbage-collection.md]: Surely, we should remove this note....

**Comment by mistyhacks** [comment]
Yea, please just remove the note instead. You can use the same PR, just change the title. :)

**Comment by makocchi-git** [comment]
@mistyhacks Thank you for your comment. I got it.

**Pull request merged**